superset-frontend/plugins/plugin-chart-echarts/src/BigNumber/index.ts

```diff
@@ -19,3 +19,4 @@
 
 export { default as BigNumberChartPlugin } from './BigNumberWithTrendline';
 export { default as BigNumberTotalChartPlugin } from './BigNumberTotal';
+export { default as PopKPIPlugin } from './BigNumberPeriodOverPeriod';
```

Comment:
Totally doable later on in a follow up PR, but could we align the component's name with it's directory name? i.e, from `PopKPIPlugin` to `BigNumberPeriodOverPeriod `

Reply:
Oh right, I just copied this one from previous file

Reply:
Done